addons/loyalty/static/src/js/loyalty_card_modal.js

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@odoo/owl';
+import { Dialog } from '@web/core/dialog/dialog';
+import publicWidget from '@web/legacy/js/public/public_widget';
+import { rpc } from '@web/core/network/rpc';
+
+publicWidget.registry.PortalLoyaltyWidget = publicWidget.Widget.extend({
+    selector: '.o_loyalty_card_modal_selector',
+    events: {
+        'click .o_loyalty_button_click': '_onPortalLoyalty',
+    },
+    showHistory: true,
+    showRewards: false,
+    async _loadData(ev){
+        let card_id = ev.currentTarget.dataset.card
+        let program = ev.currentTarget.dataset.program
```

Comment:
```suggestion
        let card_id = ev.currentTarget.dataset.card;
        let program = ev.currentTarget.dataset.program;
```
why let not const :thinking: